src/components/Admin/ListUser/ListUser.js

```diff
@@ -474,16 +528,31 @@ export default class ListUser extends Component {
                           </div>
                         </Dialog>
                       </div>
-
+                      <Search
+                        placeholder="search by email"
+                        style={{ margin: '5px 25% 20px 25%', width: '50%' }}
+                        size="default"
+                        onSearch={value => this.handleSearch(value)}
```

Comment:
Is there some event like `onChange` which filters the user as the search field is changed? This looks good too though.

Reply:
@anshumanv The data is not filtered on the front end but instead an api is called to return the desired search results so its better if we call api only on click or enter

Reply:
Okay then, works fine either way. :+1: